metadata.rb

```diff
@@ -1,18 +1,17 @@
-name             'pdns'
+name             'chef-pdns'
```

Comment:
This goes against supermarket naming conventions. `chef-` is only for cookbooks owned by `chef-` organization technically. THis needs to stay `pdns` or we can file with supermarket/chef to get `dnsimple-` space.